cypress/integration/home-page/home-page.spec.ts

```diff
@@ -0,0 +1,246 @@
+/// <reference types="cypress" />
+
+import type { Devfile, FilterDataElem } from 'custom-types'
```

Comment:
lets move tests to `tests/integration/*` and remove the keyword `cyrpress` from the project tree